a few updates for the console menu /etc/rc.initial

luckman212 opened the pull request:
- [x] Redmine Issue: https://redmine.pfsense.org/issues/13258
- [x] Ready for review

### changes
- add full pathnames to all binaries (before some were and some weren't)
- less forking for process checking, instead of ps | grep, use pgrep
- clean up spaces vs tabs
- remove the text "cheap menu"
- fixup hidden option 100 (links browser)

**n.b.** the `patch` command is failing to automatically patch this via system patches, I think because the open source version differs so much from the pfS+ version that has hardware-specific elements. So for testing, I had to manually patch my 6100 to test it. Not sure if there is a way to get it to cooperate with System Patches, maybe breaking it up into a set of smaller commits, but hopefully these changes are small enough that it can be reviewed without all that extra fuss.

luckman212 commented:
**n.b.** the dashboard may not actually load with the `links` browser ... but I was able to load other pages e.g. google. So, might still be useful to leave it in.

![image](https://user-images.githubusercontent.com/1992842/172932940-02d9dbb1-3666-4e4b-921e-a7184002d083.png)

![image](https://user-images.githubusercontent.com/1992842/172932948-a9d7580f-1298-4624-9baf-63e6a0258809.png)

jim-p commented on src/etc/rc.initial:
```diff
@@ -151,18 +148,21 @@ case ${opmode} in
+			*)     port=80;;
+		esac
 	fi
+	links "${protocol}://localhost:${port}"
```
`protocol` could be empty, so it needs an extra test to drop a default value in there if it is (`http`).

luckman212 replied:
Didn't know protocol could be empty. Ok I will rework to account for that.

luckman212 commented:
Didn't know protocol could be empty. Ok I will rework to account for that.

luckman212 commented:
Should be good now @jim-p

luckman212 commented:
Couple more very minor touchups

- removed `11 | 111)` -- why was that there? to catch typos?
- consistency with single vs double quotes
- removed a couple of stray `;`s
